doc-Monitoring_Alerts_and_Reporting/topics/To_create_chargeback_rates.adoc

```diff
@@ -1,17 +1,56 @@
 [[_to_create_chargeback_rates]]
 = Creating Chargeback Rates
 
-{product-title} allows you to create your own set of computing and storage costs.
+{product-title} allows you to create your own set of computing and storage costs to use for billing.
+
+Chargeback rates can be configured at a single rate or in tiers, where one rate is assigned to one usage range, and another rate is assigned to a different usage range. You can also assign fixed and variable rates per tier if desired.
+
+
+== Creating a Chargeback Rate
 
 . Navigate to menu:Cloud Intelligence[Chargeback].
 . Click the *Rates* accordion and select either *Compute* or *Storage*.
 
-* Use *Compute* to set chargeback rates for CPU, disk I/O, memory, network I/O, and fixed items.
-* Use *Storage* to set chargeback rates for fixed and storage items.
+* Use *Compute* to set chargeback rates for CPU, disk I/O, fixed compute cost, memory, and network I/O items.
+* Use *Storage* to set chargeback rates for fixed storage cost and disk storage.
 . Click  image:1847.png[] *(Configuration)*,  image:1862.png[] *(Add a new Chargeback Rate)* to create a new chargeback rate.
 . Type in a *Description* for the chargeback rate.
+. Select a currency.
 . For each item that you want to set, type in a rate and select a time option.
```

Comment:
@dayleparker Hi Dayle, I understand chargeback rates can be configured at a single rate or in tiers and now we have two separate procedures for creating rates. Though I was wondering if we need more clarity in this step or if the actual details are now covered in the new procedure for creating tiered rate? I might be missing something here but when I follow this procedure I see the same screen/options under Rate Details such as Description, Per Time, Per Unit, Range Start - Finish, among others which are covered in the example for creating a tiered rate. Thoughts?